### PR TITLE
Feature | Hero component

### PR DESCRIPTION
**Pull request by nmiz1987**
When there is a design, I will change the structure accordingly, including the color palette

The code is responsive, both for PC and mobile,
This is a screenshot of the final result:
image

### CODE REVIEW

File: website/src/images/hero.jpg
Comment (nir2002) [inline website/src/images/hero.jpg:1]: Is this free to use? we need to be careful with copyrights

File: website/src/components/Hero.jsx [hunk 1]
```diff
@@ -1,72 +1,43 @@
...
-        </strong>
-        , Founder at Retail Park
-      </figcaption>
-    </figure>
-  )
-}
+import coverImage from '@/images/hero.jpg'
+import Link from 'next/link'
 
 export function Hero() {
```
Comment (nir2002) [inline website/src/components/Hero.jsx]: lets use export default

File: website/src/components/Hero.jsx [hunk 2]
```diff
@@ -1,72 +1,43 @@
...
-            />
-          </div>
-          <div className="relative z-10 mx-auto flex w-64 rounded-xl bg-slate-600 shadow-xl md:w-80 lg:w-auto">
-            <Image className="w-full" src={coverImage} alt="" priority />
-          </div>
-        </div>
-        <div className="relative px-4 sm:px-6 lg:col-span-7 lg:pb-14 lg:pl-16 lg:pr-0 xl:pl-20">
-          <div className="hidden lg:absolute lg:-top-32 lg:bottom-0 lg:left-[-100vw] lg:right-[-100vw] lg:block lg:bg-slate-100" />
-          <Testimonial />
+    <>
```
Comment (nir2002) [inline website/src/components/Hero.jsx]: Lets split this to 3 components:
1. Header
2. Content
3. Link (that you already used)

leave them inside the file, no need for new component files at the moment

File: website/src/components/Hero.jsx [hunk 3]
```diff
@@ -1,72 +1,43 @@
...
+      <header className=" overflow-hidden py-10">
+        <div className="px-auto text-center">
+          <h1 className="bold text-4xl text-purple-600 sm:text-5xl">
+            Welcome to{' '}
+            <span className=" sm: block bg-gradient-to-r from-purple-600 to-blue-800 bg-clip-text text-4xl text-5xl font-extrabold uppercase tracking-wide text-transparent sm:inline">
+              craftonite
```
Comment (nir2002) [inline website/src/components/Hero.jsx]:
```suggestion
              Craftonite
```

File: website/src/components/Hero.jsx [hunk 4]
```diff
@@ -1,72 +1,43 @@
...
-        </div>
-        <div className="relative px-4 sm:px-6 lg:col-span-7 lg:pb-14 lg:pl-16 lg:pr-0 xl:pl-20">
-          <div className="hidden lg:absolute lg:-top-32 lg:bottom-0 lg:left-[-100vw] lg:right-[-100vw] lg:block lg:bg-slate-100" />
-          <Testimonial />
+    <>
+      <header className=" overflow-hidden py-10">
+        <div className="px-auto text-center">
+          <h1 className="bold text-4xl text-purple-600 sm:text-5xl">
+            Welcome to{' '}
+            <span className=" sm: block bg-gradient-to-r from-purple-600 to-blue-800 bg-clip-text text-4xl text-5xl font-extrabold uppercase tracking-wide text-transparent sm:inline">
```
Comment (nir2002) [inline website/src/components/Hero.jsx]: mobile first design:

```suggestion
            <span className="block bg-gradient-to-r from-purple-600 to-blue-800 bg-clip-text text-5xl font-extrabold uppercase tracking-wide text-transparent inline">
```

File: website/src/components/Hero.jsx [hunk 5]
```diff
@@ -1,72 +1,43 @@
...
-              </Button>
-            </div>
-          </div>
+      </header>
+      <content className="mx-auto flex w-5/6 flex-col-reverse items-center gap-3 sm:w-2/3 sm:flex-row sm:items-start sm:gap-10">
+        <Image
+          width={300}
+          height={300}
+          src={coverImage}
+          alt="craftonite"
```
Comment (nir2002) [inline website/src/components/Hero.jsx]: Craftonite logo

File: website/src/components/Hero.jsx [hunk 6]
```diff
@@ -1,72 +1,43 @@
...
+          priority
+        />
+        <div className="flex flex-col items-start gap-3 sm:gap-5">
+          <h2 className="text-center text-2xl sm:text-start sm:text-3xl ">
```
Comment (nir2002) [inline website/src/components/Hero.jsx]: you can move text-center to the div className I think

File: website/src/components/Hero.jsx [hunk 7]
```diff
@@ -1,72 +1,43 @@
...
+            Here you can find the best videos in the best price for your
+          </h2>
+          <h3 className="text-1xl text-center sm:text-start sm:text-2xl">
```
Comment (nir2002) [inline website/src/components/Hero.jsx]: you can move text-center to the div className I think

**Review by nir2002 — CHANGES_REQUESTED** [review]

File: website/src/images/hero.jpg
Comment (nmiz1987) [reply website/src/images/hero.jpg:1]: The picture is from unsplash: [link](https://unsplash.com/photos/mYBLqZzs3SM)

File: website/src/components/Hero.jsx (hunk 2 above)
Comment (nmiz1987) [reply website/src/components/Hero.jsx]: Done

File: website/src/components/Hero.jsx (hunk 1 above)
Comment (nmiz1987) [reply website/src/components/Hero.jsx]: Done

File: website/src/components/Hero.jsx (hunk 3 above)
Comment (nmiz1987) [reply website/src/components/Hero.jsx]: Done

File: website/src/components/Hero.jsx (hunk 4 above)
Comment (nmiz1987) [reply website/src/components/Hero.jsx]: Done

File: website/src/components/Hero.jsx (hunk 5 above)
Comment (nmiz1987) [reply website/src/components/Hero.jsx]: Done

File: website/src/components/Hero.jsx (hunk 6 above)
Comment (nmiz1987) [reply website/src/components/Hero.jsx]: Done

File: website/src/components/Hero.jsx (hunk 7 above)
Comment (nmiz1987) [reply website/src/components/Hero.jsx]: Done

**Comment by nmiz1987** [comment]
I wrote it as "mobile first", I had an empty space "sm: block" that confused you...
I fixed the style, a cleaner code.
Note that there are 2 .gitignore files and git is still doesn't ignore the .next folder

**Pull request closed without merging**